ecl/eclcmd/eclcmd_common.hpp

```diff
@@ -173,6 +173,8 @@ typedef IEclCommand *(*EclCommandFactory)(const char *cmdname);
 #define ECLOPT_VERBOSE "--verbose"
 #define ECLOPT_VERBOSE_S "-v"
 
+const char *queryEclccPath(bool optVerbose);
```

Comment:
picky: should really have a comment describing what it does, but fairly obvious, and only moving it.